decidim/decidim
# Fix weird behavior with votes in comments

#### :tophat: What? Why?
This PR fixes two bugs:
- When you vote a comment with a thread, the upvote was propagated down to the responses.
- When you refresh a page with a comment you voted on before, the opposite vote button was not disabled.

#### :pushpin: Related Issues
- Fixes #11320 

#### Testing
1. Sign in and go to a component with a comment with a thread.
2. Upvote a comment with a thread and see it only upvotes the comment and it's not propagated down to the replies.
3. Refresh the page and see the downvote button is disabled.

:hearts: Thank you!
